WcaOnRails/config/locales/en.yml

```diff
@@ -214,6 +214,9 @@ en:
         contact: "Contact"
         information: "Information"
         use_wca_registration: "I would like to use the WCA website for registration"
+        competitor_limit_flag: "I would like to specifiy a limit on the number of competitors that can enter this competition."
```

Comment:
I think this will surprise people when they find out it doesn't affect the registration flow. Could you add a sentence to this explaining the behavior of this (and that the WST is working on adding explicit support for this to the registration flow, but it requires some other work first).